From 4b2603ba12077bedfab2d0f83e8e5f6becfa0081 Mon Sep 17 00:00:00 2001
From: Ilya Ledvich <ilya@compulab.co.il>
Date: Fri, 15 Jul 2022 16:07:58 +0300
Subject: [PATCH] compulab: iot-gate-imx8plus: change rtc devices enumeration

Change rtc devices enumeration to cause external AB1805 RTC to be
the default one.

Signed-off-by: Ilya Ledvich <ilya@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/sb-iotgimx8plus.dtsi | 6 ++++++
 arch/arm64/boot/dts/compulab/ucm-imx8m-plus.dtsi  | 2 +-
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/compulab/sb-iotgimx8plus.dtsi b/arch/arm64/boot/dts/compulab/sb-iotgimx8plus.dtsi
index d9c7afca2b99..626bec53227d 100644
--- a/arch/arm64/boot/dts/compulab/sb-iotgimx8plus.dtsi
+++ b/arch/arm64/boot/dts/compulab/sb-iotgimx8plus.dtsi
@@ -6,6 +6,12 @@
 #include <dt-bindings/leds/common.h>
 
 / {
+
+	aliases {
+		rtc0 = &ab1805_rtc;
+		rtc1 = &snvs_rtc;
+	};
+
 	chosen {
 		stdout-path = &uart2; /* console */
 	};
diff --git a/arch/arm64/boot/dts/compulab/ucm-imx8m-plus.dtsi b/arch/arm64/boot/dts/compulab/ucm-imx8m-plus.dtsi
index 460f8e4cfa10..1e22741d0a6e 100644
--- a/arch/arm64/boot/dts/compulab/ucm-imx8m-plus.dtsi
+++ b/arch/arm64/boot/dts/compulab/ucm-imx8m-plus.dtsi
@@ -290,7 +290,7 @@
 		pagesize = <16>;
 	};
 
-	rtc@69 {
+	ab1805_rtc: rtc@69 {
 		compatible = "ab1805";
 		reg = <0x69>;
 		pagesize = <16>;
-- 
2.17.1

